apps/admin/frontend/src/components/reporting/filter_editor.tsx

```diff
@@ -281,7 +281,7 @@ export function FilterEditor({
               }}
               ariaLabel="Edit Filter Type"
             />
-            <Predicate>equals</Predicate>
+            <Predicate>is</Predicate>
```

Comment:
These images incorporate a number of changes:

<img width="1147" alt="Screen Shot 2024-10-09 at 3 20 07 PM" src="https://github.com/user-attachments/assets/86163581-195a-4a5a-931d-5eab8205629b">
<img width="1154" alt="Screen Shot 2024-10-09 at 3 17 13 PM" src="https://github.com/user-attachments/assets/19472870-9dc8-4dc4-937e-98a38aebd6e5">